### PR TITLE
Account/wishlist test ready for review

### LINKED ISSUE #15: Write feature tests for AccountController
Test that:
- GET /wishlist returns a list of items successfully
- GET /wishlist/{id} returns an item by id successfully
- GET /wishlist/{id} where ID does not exist returns a 404
- aim for 100% code coverage on AccountController

### PR DESCRIPTION
**Pull request by mdigrandi**
resolves #15 - ready for review/ any additional notes

### CODE REVIEW

File: tests/Feature/accountControllerTest.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        parent::setUp();
+    }
+
+    public function test_get_wishlist_returns_ok()
+    {
+        $response = $this->get('/wishlist');
+        $response->assertStatus(200);
+    }
+
+    // public function test_get_wishlist_by_id_returns_ok()
```
Comment (12bitsample) [inline tests/Feature/accountControllerTest.php:26]: Great job! Only suggestion I might have is to delete the commented out stuff. I tend to forget to do that all the time, haha. Keep up the good work!

**Review by 12bitsample — APPROVED** [review]
Everything looks good to me, great job!

File: tests/Feature/accountControllerTest.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+    }
+
+    public function test_get_wishlist_returns_ok()
+    {
+        $response = $this->get('/wishlist');
+        $response->assertStatus(200);
```
Comment (oldskoolfan) [inline tests/Feature/accountControllerTest.php:23]: you should be testing the view model using `$response->assertViewHas()`. also, where are the other tests:
GET /wishlist/{id} returns an item by id successfully
GET /wishlist/{id} where ID does not exist returns a 404

Comment (mdigrandi) [reply tests/Feature/accountControllerTest.php:23]: In the most recent Wishlist controller I didn't see an ID field so I had that test commented out

Comment (mdigrandi) [reply tests/Feature/accountControllerTest.php:23]: $response->assertViewHas('wishlist', $............); <not sure what data would go in this field, I tried $wishlist or $title and neither one was working

**Comment by oldskoolfan** [comment]
Unfortunately mike already created tests for AccountController in PR #20 , i'm sorry this was missed and you worked on this for nothing :(

**Pull request closed without merging**